app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationCollection/integrationCollectionItem/integrationCollectionItem.jsx

```diff
@@ -91,6 +92,14 @@ IntegrationCollectionItem.propTypes = {
   creator: PropTypes.string,
   disabled: PropTypes.bool,
   openIntegration: PropTypes.func,
+  item: PropTypes.shape({
```

Comment:
now it looks fine. And now we can see that in item we have creator and title, and we can extract it from item.